extension/entitystore/ec2Info_test.go

```diff
@@ -229,8 +200,10 @@ func TestLogMessageDoesNotIncludeResourceInfo(t *testing.T) {
 
 			logOutput := buf.String()
 			log.Println(logOutput)
-			assert.NotContains(t, logOutput, ei.InstanceID)
-			assert.NotContains(t, logOutput, ei.AutoScalingGroup)
+			ei.mutex.RLock()
```

Comment:
nit: This won't do anything. The `GetInstanceID` and `GetAutoScalingGroup` already acquire a read lock.